images/builder-golang/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/library/golang:1.20-alpine3.17
+
+ENV GOLANGCI_LINT_CACHE $GOPATH/cache
+ENV GOCACHE $GOPATH/cache
+
+SHELL ["/bin/ash","-c"]
+WORKDIR /usr/work
+LABEL org.opencontainers.image.source="https://github.com/dyrector-io/dyrectorio/tree/develop/images/alpine-proto"
+
+RUN mkdir -p $GOPATH/src && \
+    mkdir -p $GOPATH/bin && \
+    mkdir -p $GOPATH/cache && \
+    mkdir -p $GOPATH/pkg
+
+RUN apk add alpine-sdk docker git zstd bash
+
+# Need these for integration tests
```

Comment:
The naming is a bit misleading then, they are also in use as a test runtime, they do more than _build_. Unfortunately if we would want to split container runtime to better isolate concerns (builder/test runtime images), we would be without golang that is used to run tests.

Reply:
We use these images in our build process(testing is part of the build process), so they got a name like this. If you have something on your mind please tell, I'm open to suggestions.

Reply:
We can live with this, I suppose.